neutron/tests/functional/agent/l2/extensions/test_ovs_agent_qos_extension.py

```diff
@@ -107,6 +109,18 @@ def wait_until_bandwidth_limit_rule_applied(self, port, rule):
         l2_extensions.wait_until_bandwidth_limit_rule_applied(
             self.agent.int_br, port['vif_name'], rule)
 
+    def _assert_dscp_marking_rule_is_set(self, qos_policy, dscp_tag):
+        current_dscp_tag = self.agent.int_br.get_dscp_marking_rule(qos_policy)
```

Comment:
suggest: current_dscp_tag = self.agent.int_br.get_dscp_mark(port['ofport'])

the argument here, port['ofport'],  might instead be port.ofport or something else; i'll try to figure this out if you don't readily know.